### PR TITLE
Update intro with Jellyfish recommendations

### PR DESCRIPTION
**Pull request by ReneeRendon**
Update wording and add product links

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-73d926db4c/how-to/)

File: content/cloud-dns/modify-your-hosts-file.md [hunk 1]
```diff
@@ -5,22 +5,20 @@ title: Modify your hosts file
...
 product: Cloud DNS
 product_url: cloud-dns
 ---
 
-Modifying your **hosts** file enables you to override the domain name system
-(DNS) for a domain on a specific machine. This process is useful when you
-want to perform the following tasks:
+Modifying your hosts file enables you to override the domain name system
+(DNS) for a domain on a specific machine. [DNS management](https://www.rackspace.com/cloud/dns) is useful when you
+want to perform test your site without the test link prior to going live with SSL,
```
Comment (RackerKate) [inline content/cloud-dns/modify-your-hosts-file.md]: I think there's something amiss with "perform test" here.

File: content/cloud-dns/modify-your-hosts-file.md [hunk 2]
```diff
@@ -5,22 +5,20 @@ title: Modify your hosts file
...
 
-- Test your site without the test link prior to going live with SSL.
-- Verify that an alias site works prior to DNS changes.
-- Work on other DNS-related tasks.
-
-Modifying your **hosts** file causes your local machine to look directly at
-the Internet Protocol (IP) address that you specify.
+Modifying your hosts file causes your local machine to look directly at
+the Internet Protocol (IP) address that you specify. Rackspace offers 
+[managed hosting solutions](https://www.rackspace.com/managed-hosting) to assist the handling of these resources.
```
Comment (RackerKate) [inline content/cloud-dns/modify-your-hosts-file.md]: "To assist with..."?

File: content/cloud-dns/modify-your-hosts-file.md [hunk 3]
```diff
@@ -47,8 +45,9 @@ these entries.
 
 ### Windows
 
-Windows 10, Windows 8, Windows 7, and Windows Vista use User Account Control
-(UAC). As a result, you must run Microsoft Notepad as an administrator.
+If you are using a [Microsoft operating system](https://www.rackspace.com/microsoft) like Windows 10, Windows 8,
```
Comment (RackerKate) [inline content/cloud-dns/modify-your-hosts-file.md]: It looks like there might be an extra space after the close parens.

File: content/cloud-dns/modify-your-hosts-file.md [hunk 4]
```diff
@@ -47,8 +45,9 @@ these entries.
 
 ### Windows
 
-Windows 10, Windows 8, Windows 7, and Windows Vista use User Account Control
-(UAC). As a result, you must run Microsoft Notepad as an administrator.
+If you are using a [Microsoft operating system](https://www.rackspace.com/microsoft) like Windows 10, Windows 8, 
+Windows 7, and Windows Vista use User Account Control (UAC). As a result,
```
Comment (RackerKate) [inline content/cloud-dns/modify-your-hosts-file.md]: I think there's something amiss with the second part of this sentence.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6164c7bdc8/how-to/)

File: content/cloud-dns/modify-your-hosts-file.md [hunk 5]
```diff
@@ -47,8 +46,9 @@ these entries.
 
 ### Windows
 
-Windows 10, Windows 8, Windows 7, and Windows Vista use User Account Control
-(UAC). As a result, you must run Microsoft Notepad as an administrator.
+If you are using a [Microsoft operating system](https://www.rackspace.com/microsoft) 
+like Windows 10, Windows 8, Windows 7, and Windows Vista use User Account Control (UAC).
```
Comment (RackerKate) [inline content/cloud-dns/modify-your-hosts-file.md]: Does this mean that if the user is running one of these Windows OS, they need to use User Account Control (UAC)? Or is it saying that Windows uses UAC, and that as a result (of that), you have to run Notepad as an admin?

**Pull request merged**